### PR TITLE
,,,=^0.0^=,,,

### PR DESCRIPTION
**Pull request by boujeepossum**

### CODE REVIEW

**Comment by scsper** [comment]
kya bawkwas eh?

**Comment by scsper** [comment]
do you want me to merge this?  i'm fixing the build.

File: gulp_tasks/test.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+var gulp = require('gulp');
+var mocha = require('gulp-mocha');
+
+gulp.task('test', function() {
+    gulp.src(['test/setup/unit.js', 'src/**/_unit_/*_test.js'])
+        .pipe(mocha({reporter: 'nyan'}));
```
Comment (scsper) [inline gulp_tasks/test.js]: lol spec please

**Comment by scsper** [comment]
You have to change travis.yml to run gulp now

File: gulp_tasks/webpack.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+var gulp = require('gulp');
+var webpack = require('webpack');
+var gutil = require('gulp-util');
+
+gulp.task('webpack', function(callback) {
```
Comment (scsper) [inline gulp_tasks/webpack.js:5]: if you're not going to use webpack_config then get rid of it

File: gulp_tasks/test.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+var gulp = require('gulp');
+var mocha = require('gulp-mocha');
+
+gulp.task('test', function() {
+    gulp.src(['test/setup/unit.js', 'src/**/_unit_/*_test.js'])
```
Comment (scsper) [inline gulp_tasks/test.js:5]: you got rid of test/*_/_ but you left this line in.  are we going to keep chai / sinon chai / mockery / nock?

Comment (boujeepossum) [inline gulp_tasks/test.js:5]: the setup will still be there, but the tools need to be configured later

**Comment by boujeepossum** [comment]
Addressed / rebased

**Pull request merged**